### ISSUE TITLE
CreateServiceFabricCluster fails if directory contains space in a name

### ISSUE THREAD

**Issue by artisticcheese** [microsoft/service-fabric#752]
Code below which is part of CreateServiceFabricCluster will fail if path contains spaces. 

```
 $DCExtractArguments = "/E /Y /L `"$ThisScriptPath`""
    $DCExtractOutput = cmd.exe /c "$DCAutoExtractorPath $DCExtractArguments && exit 0 || exit 1"
    if($LASTEXITCODE -eq 1)
    {
        Write-Host "Extracting DeploymentComponents Cab ran into an issue."
        Write-Host $DCExtractOutput
        exit 1
    }
```
Error

```
DBG]: PS C:\gd\Documents\docker\Windows Containers\sf> cmd.exe /c "$DCAutoExtractorPath $DCExtractArguments && exit 0 || exit 1"
cmd.exe : 'C:\gd\Documents\docker\Windows' is not recognized as an internal or external command,
At line:1 char:1
```

Need to put quotes around `$DCAutoExtractorPath`

**Comment by moldyseaswimmer** [comment]
add this before line 43 
$DCAutoExtractorPath = "`"$DCAutoExtractorPath`""

**Comment by maburlik** [comment]
Created tracking item for this. Thanks for reporting.

**Comment by maburlik** [comment]
This affects
AddNode.ps1
CreateServiceFabricCluster.ps1
DownloadServiceFabricRuntimePackage.ps1
RemoveNode.ps1
RemoveServiceFabricCluster.ps1
TestConfiguration.ps1

Aiming to have this included in the 7.2 release.
